ruby/rubocop.yml

```diff
@@ -13,26 +16,141 @@ AllCops:
 Metrics/LineLength:
   Max: 120
 
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
```

Comment:
Vill vi inte ha en gräns för storlek på klasser?

Reply:
Whops, denna skulle in i våran extern branch, inte master.

Vi har stängt av den copen då vi känner att det är svårt att säga att en klass ska ha en specifik max längd, och att det ibland bara blir ett brus med felen.
Som oftast extraherar man en klass eller annat när man känner att det behövs, så det hänger lite på mer på den människan som skriver koden eller som gör CR att ställa en sån fråga.